cpp/benchmarks/ast/polynomials.cpp

```diff
@@ -30,12 +30,26 @@
 
 #include <random>
 
+enum class engine_type : uint8_t { AST = 0, JIT = 1 };
+
+static engine_type engine_from_string(std::string_view str)
+{
+  if (str == "ast") {
+    return engine_type::AST;
+  } else if (str == "jit") {
+    return engine_type::JIT;
+  } else {
+    CUDF_FAIL("unrecognized engine enum: " + std::string(str));
+  }
+}
+
 template <typename key_type>
 static void BM_ast_polynomials(nvbench::state& state)
 {
   auto const num_rows         = static_cast<cudf::size_type>(state.get_int64("num_rows"));
   auto const order            = static_cast<cudf::size_type>(state.get_int64("order"));
   auto const null_probability = state.get_float64("null_probability");
+  engine_type engine          = engine_from_string(state.get_string("engine"));
```

Comment:
Nit: just to be consistent with t he lines above it.
```suggestion
  auto const engine          = engine_from_string(state.get_string("engine"));
```